dashboard/lib/metrics/events.rb

```diff
@@ -4,8 +4,9 @@ module Metrics
   module Events
     class << self
       # Logs an event, delegating to the appropriate handler based on environment
-      def log_event(user:, event_name:, event_value: nil, metadata: {}, enabled_experiments: nil)
+      def log_event(user:, event_name:, event_value: nil, metadata: {}, get_enabled_experiments: false)
```

Comment:
Thank you for adding this! And makes sense that it can be optional to save the query. I wonder if we could cache this somehow (but that can be a future consideration)!

Reply:
Sounds good, yeah caching could be an interesting thing to explore 👍. We evaluate later to see how many event metrics are part of an experiment vs. how many aren't, and dial this in to favor the majority as it makes sense.

Reply:
I think the idea with this configuration isn't necessarily about *only* seeing users inside experiments: it's about viewing *whether or not the user is in experiments. Perhaps @dmcavoy could provide more context here. My instinct is that we *always want experiment data instead of having to opt in.

Reply:
I'm so sorry for all the churn on this!! I just spoke with @bethanyaconnor offline about this and I think our implementation of experiments is messy enough between the backend and front end to keep this as an opt-in to save the additional query. Resolving!